client/js/components/SpeakerCard/SpeakerCard.js

```diff
@@ -4,8 +4,17 @@ import styles from './styles';
 import {TouchableOpacity} from 'react-native-gesture-handler';
 import Text from '../CustomText/CustomText';
 
-const SpeakerCard = ({name, style, source}) => {
-  return (
+const SpeakerCard = ({name, style, source, route}) => {
```

Comment:
your props can be just one object.

Reply:
This is just dummy data for now, can we reconfigure props when we start working on the db queries?